tests/saitests/py3/sai_qos_tests.py

```diff
@@ -4645,6 +4654,11 @@ def runTest(self):
             if hwsku == 'DellEMC-Z9332f-O32' or hwsku == 'DellEMC-Z9332f-M-O16C64':
                 send_packet(self, src_port_id, pkt, (pkts_num_egr_mem +
                                                      pkts_num_leak_out + pkts_num_trig_pfc) // cell_occupancy - margin)
+            elif 'cisco-8000' in asic_type:
+                queue = pg
+                fill_leakout_plus_one(self, src_port_id, dst_port_id, pkt, queue, asic_type)
+                send_packet(self, src_port_id, pkt, (pkts_num_leak_out +
```

Comment:
as I motioned before,
if we use  below sequence:

```
send_packet(self, src_port_id, pkt, (pkts_num_leak_out + pkts_num_trig_pfc) // cell_occupancy - margin - 1)
dynamically_compensate_leakout( ... ... )
```

it can reduce number of compensatation leaks.
Or are there other benefits to the above Cisco approach?